README.md

```diff
@@ -16,7 +16,7 @@ Two ports of the NIC under test are directly connected to the traffic generator
 On the DUT server, the following RPM packages are required,
 * tmux
 * nmap
-* podman
+* a container manager (podman or docker)
```

Comment:
Maybe put a recommendation to use podman  for RHEL and how to install podman if it is missing, e.g. 'yum -y install podman'. For docker it is out of the documentation scope.

Reply:
Added this to the README in the most recent commit